### PR TITLE
Develop nutty modify framework for315 modifymethodname

### PR DESCRIPTION
**Pull request by nutty898**
Hide "addNode"

### CODE REVIEW

File: cocos/ui/UIListView.h [hunk 1]
```diff
@@ -182,11 +182,10 @@ class ListView : public ScrollView
     
     virtual void removeAllChildren() override{removeAllChildrenWithCleanup(true);};
     virtual void removeAllChildrenWithCleanup(bool cleanup) override {ScrollView::removeAllChildrenWithCleanup(cleanup);};
-    virtual Vector<Node*>& getChildren() override{return ScrollView::getChildren();};
-    virtual const Vector<Node*>& getChildren() const override{return ScrollView::getChildren();};
+    virtual Vector<Widget*>& getWidgets() override{return ScrollView::getWidgets();};
+    virtual const Vector<Widget*>& getWidgets() const override{return ScrollView::getWidgets();};
```
Comment (dumganhar) [inline cocos/ui/UIListView.h:186]: If ListView::getWidget just return its parent's `getWidget`, we don't need to override this function.

File: cocos/ui/UIWidget.cpp [hunk 2]
```diff
@@ -149,31 +156,19 @@ void Widget::sortAllChildren()
...
 {
-    CCASSERT( aTag != Node::INVALID_TAG, "Invalid tag");
-
-    for (auto& child : _widgetChildren)
-    {
-        if(child && child->getTag() == aTag)
-            return child;
-    }
-    return nullptr;
+    return Node::getChildByTag(aTag);
```
Comment (dumganhar) [inline cocos/ui/UIWidget.cpp]: Don't need to override this function since you just return Node::getChildByTag.

File: cocos/ui/UIListView.h (hunk 1 above)
Comment (nutty898) [inline cocos/ui/UIListView.h:186]: I don't want users call those methods, so I override those function and make them be protected.

File: cocos/ui/UIWidget.cpp (hunk 2 above)
Comment (nutty898) [inline cocos/ui/UIWidget.cpp]: Alright, I'll remove the codes.

**Comment by cocosQA** [comment]
Dear nutty898:
  We have created a new branch 'v3' to replace branch 'develop', and this PR has been moved from 'develop' branch to 'v3' branch, the new PR is https://github.com/cocos2d/cocos2d-x/pull/6446
The old 'develop' branch will be deleted soon, and this PR will be closed accordingly.

**Pull request closed without merging**